packages/online-playground/src/App.vue

```diff
@@ -19,15 +19,47 @@ if (hash.startsWith('__DEV__')) {
   useDevMode.value = true
 }
 
-// TODO: we should fetch the latest version and set it by default here
-const store = new ReplStore({
-  serializedState: hash,
-  defaultVueRuntimeURL:
-    'https://cdn.jsdelivr.net/npm/@vue/runtime-dom@3.4.21/dist/runtime-dom.esm-browser.js',
-  defaultVueServerRendererURL:
-    'https://cdn.jsdelivr.net/npm/@vue/server-renderer@3.4.21/dist/server-renderer.esm-browser.js',
+const {
+  importMap: builtinImportMap,
+  vueVersion,
+  productionMode,
+} = useVueImportMap({
+  runtimeDev: 'https://cdn.jsdelivr.net/npm/vue@3/dist/vue.runtime.esm-browser.js',
+  runtimeProd: 'https://cdn.jsdelivr.net/npm/vue@3/dist/vue.runtime.esm-browser.prod.js',
+  serverRenderer: 'https://cdn.jsdelivr.net/npm/@vue/server-renderer@3/dist/server-renderer.esm-browser.js',
 })
 
+const store = useStore(
+  {
+    // pre-set import map
+    builtinImportMap: computed(() => ({
+      imports: {
+        ...builtinImportMap.value.imports,
+        ...(import.meta.env.PROD
+        ? {
+            pinia: '/pinia.esm-browser.js',
+            '@vue/devtools-api': 'https://cdn.jsdelivr.net/npm/@vue/devtools-api@6.6.1/lib/esm/index.js',
+            // Latest 3.x version
+            'vue-demi': 'https://cdn.jsdelivr.net/npm/vue-demi@0/lib/v3/index.mjs',
+            // Latest 5.x version
+            'typescript': 'https://cdn.jsdelivr.net/npm/typescript@5/lib/typescript.min.js',
+          }
+        : {
+            pinia: '/src/pinia-dev-proxy',
+            vue: '/src/vue-dev-proxy',
+            'vue/server-renderer': '/src/vue-server-renderer-dev-proxy',
+            // Latest 5.x version
+            'typescript': 'https://cdn.jsdelivr.net/npm/typescript@5/lib/typescript.min.js',
```

Comment:
We shouldn't need typescript, the [vue js playground doesn't have it](https://play.vuejs.org)